fixtures/attribute-behavior/AttributeTableSnapshot.md

```diff
@@ -701,23 +701,23 @@
 ## `as` (on `<div>`)
 | Test Case | Flags | Result |
 | --- | --- | --- |
-| `as=(string)`| (changed)| `"a string"` |
+| `as=(string)`| (initial)| `<empty string>` |
```

Comment:
This is an unrelated change, and seems like Chrome version difference. Setting `<link as="whatever">` doesn't return `"whatever"` from `link.as` property anymore. But it works with valid values like `"audio"`.